spec/features/logged_in_user_can_see_dashboard_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.feature "logged in user can view dashbaord" do
+  scenario "they view user info and trips/listings" do
+    user = create(:user)
+    login(user)
+    order = create(:order)
+    User.orders.create(order)
+    click_on user.name
+    click_on "Dashboard"
+
+    expect(current_path).to eq "/dashboard"
+
+    click_on "My Trips"
+
+    expect(current_path).to eq orders_path
+  end
```

Comment:
Lets make this test more robust by including specific details that should be included for a given Reservation. This may be subject to change depending on design choices for displaying these individual reservations.

For now, it'd be nice to see an expectation on the number of orders present.